netbox_agent/lshw.py

```diff
@@ -88,22 +88,25 @@ def find_storage(self, obj):
                 self.disks.append(d)
 
         elif "nvme" in obj["configuration"]["driver"]:
-            nvme = json.loads(
-                subprocess.check_output(
-                    ["nvme", '-list', '-o', 'json'],
-                    encoding='utf8')
-            )
-
-            for device in nvme["Devices"]:
-                d = {}
-                d['logicalname'] = device["DevicePath"]
-                d['product'] = device["ModelNumber"]
-                d['serial'] = device["SerialNumber"]
-                d["version"] = device["Firmware"]
-                d['size'] = device["UsedSize"]
-                d['description'] = "NVME Disk"
-
-                self.disks.append(d)
+            try:
+                nvme = json.loads(
+                    subprocess.check_output(
+                        ["nvme", '-list', '-o', 'json'],
+                        encoding='utf8')
+                )
+
+                for device in nvme["Devices"]:
+                    d = {}
+                    d['logicalname'] = device["DevicePath"]
+                    d['product'] = device["ModelNumber"]
+                    d['serial'] = device["SerialNumber"]
+                    d["version"] = device["Firmware"]
+                    d['size'] = device["UsedSize"]
+                    d['description'] = "NVME Disk"
+
+                    self.disks.append(d)
+            except Exception:
+                pass
```

Comment:
Can we log in warning that the binary nvme isn't available and we're missing a feature ?
If we don't do that, the user may never now